### ISSUE TITLE
Keyboard behavior for navigation list

### ISSUE THREAD

**Issue by zelliott** [w3c/aria-practices#1035]
### Scenario

Suppose I have the following navigation list. The list is always visible, and each link in the list navigates to a different page in the app.

```
<nav>
  <ul>
    <li><a href="#foo">foo</a></li>
    <li><a href="#bar">bar</a></li>
    <!-- 10+ more links... -->
  </ul>
</nav>
```

### Problem

Suppose there are **a lot of links in this list**... over a dozen links. I don't like having a tab-stop for each link. Sure, AT users can use landmarks to skip over the navigation landmark, and a "skip-link" can be implemented to jump to the main content, but it's still painful.

For example, suppose a non-AT user has focus within the navigation list. They can't easily quickly jump out of the list, instead, they have to **tab through a dozen items** to get out of the navigation. This seems like bad keyboarding design.

### Potential solution

I'd to implement this navigation list with a single tab stop, and arrow-key navigation between links. However, I'm not sure if this is conventional keyboard behavior for a navigation list. That is, I only want to implement this keyboard behavior if users will know to expect it **given the fact it is a navigation list**.

I can find expected keyboard behavior for a menu here (https://www.w3.org/TR/wai-aria-practices/#menu), but I can't find any equivalent expected behavior for a navigation list. My question is-- is this a reasonable to implement "menu-like" keyboarding behavior (e.g. single tab stop, arrow key navigation) for a navigation list?

One "similar" example I found is the navigation bar on https://www.usa.gov. They have the keyboard behavior I am looking to implement (their navigation bar is structured essentially identically to the example I gave above... albeit it has nested links).

Thanks--

**Comment by carmacleod** [comment]
Something similar is in the works: https://github.com/w3c/aria-practices/issues/1028

FYI, "tabs" vs. "arrows" in navigation lists has had some discussion before, if you are interested: https://github.com/w3c/aria-practices/issues/353

**Comment by zelliott** [comment]
Thanks @carmacleod, I read through both those issues yesterday in an attempt to find an answer to my question... there's a lot to unpack esp in #353.

**Comment by carmacleod** [comment]
> there's a lot to unpack esp in #353.

Yep. Actually, thinking about it a little more, a grid might be what you want.
Have a look at Example 1 on this [APG Grid Example](http://w3c.github.io/aria-practices/examples/grid/LayoutGrids.html) page.
